.travis.yml

```diff
@@ -17,9 +17,6 @@ before_deploy:
 
 after_success:
   - Rscript -e 'covr::codecov()'
-
```

Comment:
I don't think we use any of this functionality either how, hence it is safe to remove